src/CONST.js

```diff
@@ -455,7 +455,7 @@ const CONST = {
 
     EMOJI_FREQUENT_ROW_COUNT: 3,
 
-    EMOJI_INVISIBLE_CODEPOINTS: ['fe0f', '200d'],
+    EMOJI_INVISIBLE_CODEPOINTS: ['fe0f', '200d', '2066'],
```

Comment:
Now that we are adding LTR unicode here, I think it would be better to rename it to just `INVISIBLE_CODEPOINTS`